src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -1778,11 +1804,26 @@ std::vector<std::shared_ptr<const ov::Node>> GNAPlugin::GetOutputs() {
 }
 
 InferenceEngine::QueryNetworkResult GNAPlugin::QueryNetwork(const InferenceEngine::CNNNetwork& network,
-                                                            const std::map<std::string, std::string>& config) const {
+                                                            const std::map<std::string, std::string>& config_map) const {
     InferenceEngine::QueryNetworkResult res;
 
-    if (network.getFunction()) {
-        IE_THROW(NotImplemented) << " ngraph::Function is not supported natively";
```

Comment:
suggest to move this implementation to separate method e.g.
`QueryNetworkUsingNgraph` or `QueryNetworkUsingModel`

Reply:
> suggest to move this implementation to separate method e.g. `QueryNetworkUsingNgraph` or `QueryNetworkUsingModel`

Don't see it sense to do it. It is just couple of lines.